app/pages/learn/protocols/balancer.mdx

```diff
@@ -0,0 +1,432 @@
+  | `request.minAmountsOut`     | -                    |
+  | `request.userData`          | -                    |
+  | `request.toInternalBalance` | -                    |
+
+### Target: "Composable Pools"
+
+<Callout type="info">
+  "Composable Pools" have at least one BPT in their `Pool` tokens.
+</Callout>
+
+For targets that are "Composable Pools", the following functions are allowed:
+
+#### `TOKEN · approve`
+
+Approve `Vault` to spend `TOKEN` tokens on behalf of the avatar.
+
+- **Contract:** `TOKEN`
+- **Function:** `approve(address,uint256)`
+
+  | Parameter | Constraint       |
+  | :-------- | :--------------- |
+  | `spender` | equal to `Vault` |
+  | `amount`  | -                |
+
+#### `Vault · joinPool`
+
+Deposit `TOKEN` tokens to the corresponding Balancer pool and receive `TARGET_BPT` tokens in exchange.
+
+- **Contract:** `Vault`
+- **Function:** `joinPool(bytes32,address,address,(address[],uint256[],bytes,bool))`
+
+  | Parameter                     | Constraint           |
+  | :---------------------------- | :------------------- |
+  | `poolId`                      | equal to `TARGET_ID` |
+  | `sender`                      | equal to `AVATAR`    |
+  | `recipient`                   | equal to `AVATAR`    |
+  | `request.assets`              | -                    |
+  | `request.maxAmountsIn`        | -                    |
+  | `request.userData`            | -                    |
+  | `request.fromInternalBalance` | -                    |
+  | `send`                        | equal to `true`      |
+
+#### `Vault · exitPool`
+
+Redeem `TARGET_BPT` tokens to the corresponding Balancer pool and receive `TOKEN` tokens in exchange.
+
+- **Contract:** `Vault`
+- **Function:** `exitPool(bytes32,address,address,(address[],uint256[],bytes,bool))`
+
+  | Parameter                   | Constraint           |
+  | :-------------------------- | :------------------- |
+  | `poolId`                    | equal to `TARGET_ID` |
+  | `sender`                    | equal to `AVATAR`    |
+  | `recipient`                 | equal to `AVATAR`    |
+  | `request.assets`            | -                    |
+  | `request.minAmountsOut`     | -                    |
+  | `request.userData`          | -                    |
+  | `request.toInternalBalance` | -                    |
+
+#### `Vault · setRelayerApproval`
+
+One-time (and persistent) approval of the Relayer as single-point of entry.
+
+- **Contract:** `Vault`
+- **Function:** `setRelayerApproval(address,address,bool)`
+
+  | Parameter  | Constraint         |
+  | :--------- | :----------------- |
+  | `sender`   | equal to `AVATAR`  |
+  | `relayer`  | equal to `Relayer` |
+  | `approved` | -                  |
+
+#### `Relayer · multicall`
+
+When using the `Relayer` as a single point of entry, all individual function calls to it must be wrapped within the metafunction `multicall(bytes[])`.
+
+- **Contract:** `Relayer`
+- **Function:** `multicall(bytes[])`
```

Comment:
```suggestion
- **Function:** `multicall(bytes[])`  Ξ allow sending Ether
```

Reply:
let's make it clear that `send` is not a function parameter but a special call mode to the function (where `value` is greater 0)